src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -96,8 +97,23 @@ const getId = ({ pageType, service, variant, env }: GetIdProps) => {
       getIdFunction = () => pageType;
       break;
     case LIVE_PAGE:
+      getIdFunction = (path: string) => {
+        const isTipoId = isTipoIdCheck(path);
+        const isCpsId = isCpsIdCheck(path);
```

Comment:
Do we need to know if the ID is a CPS id? Could we assume that we know we're rendering a live page, and therefore if it's not a Tipo ID, then it must be a CPS one?

If so, it means that this logic could be greatly simplified? 

Also, it would mean that there would be no need to "reconstruct" the ID (to be /service/live/id), because it is already the same as the path 

e.g. in pseudo code:
```
if isTipoId
 return id
else 
 return path
```

Reply:
We decided to keep the CPS id check because we need to return null in the LIVE_PAGE case for the test that check an error message for when the ID is not a valid tipo or cps one.

Reply:
OK that's fair enough. Technically routing in Belfrage would offer us this extra protection, but there's no harm in keeping it for now.